prereq_tools/base.py

```diff
@@ -513,11 +512,15 @@ def get(self, subdir, **kw):
             if self.__dry_run:
                 print 'Would unpack gziped tar file: %s' % basename
                 return
-            tfile = tarfile.open(basename, 'r:gz')
-            members = tfile.getnames()
-            prefix = os.path.commonprefix(members)
-            tfile.extractall()
-            if not RUNNER.run_commands(['mv %s %s' % (prefix, subdir)]):
+            try:
+                tfile = tarfile.open(basename, 'r:gz')
+                members = tfile.getnames()
+                prefix = os.path.commonprefix(members)
+                tfile.extractall()
+                if not RUNNER.run_commands(['mv %s %s' % (prefix, subdir)]):
+                    raise ExtractionError(subdir)
+            except IOError, tarfile.TarError:
```

Comment:
(pylint-redefine-in-handler) Redefining name 'TarError' from object 'tarfile' in exception handler